spring-xd-dirt/src/test/java/org/springframework/xd/dirt/module/ResourceModuleRegistryLogicTests.java

```diff
@@ -18,26 +18,33 @@
 
 package org.springframework.xd.dirt.module;
 
-import static org.hamcrest.Matchers.contains;
-import static org.junit.Assert.assertThat;
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.*;
+import static org.springframework.xd.module.ModuleType.*;
```

Comment:
If I'm not mistaken (IDEA prefs being such a mess to understand, at least for me), I'm using the Spring Core settings.
Which we should put under git somehow, by the way, so that we all use the same.